### PR TITLE
Added downloading languages

### PR DESCRIPTION
**Pull request by poradz**
image
image

### CODE REVIEW

File: layouts/basic/modules/Settings/LangManagement/LangList.tpl [hunk 1]
```diff
@@ -2,37 +2,53 @@
...
-	<tbody>
+		</thead>
+		<tbody>
 		{foreach from=App\Language::getAll(false, true) item=LANG key=ID}
-			<tr data-prefix="{$LANG['prefix']}">
-				<td>{$LANG['label']}</td>
+			<tr class="js-lang-row" data-prefix="{$LANG['prefix']}">
 				<td>{$LANG['name']}</td>
 				<td>{$LANG['prefix']}</td>
+				<td class="js-last-update" data-js="html">{$LANG['lastupdated']}</td>
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/LangManagement/LangList.tpl]:
```suggestion
				<td class="js-last-update" data-js="html">{\App\Fields\DateTime::formatToViewDate($LANG['lastupdated'])}</td>
```

File: layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{strip}
+	<div class="tpl-Settings-YetiForce-DownloadLanguageModal modal-body">
+		<table class="table mb-0 mx-auto u-w-fit">
+			{assign var="INDEX" value=0}
+			{if $LANGUAGES}
+				{foreach key=FIELD_NAME item=FIELD_DETAILS from=$LANGUAGES}
+					{if !$FIELD_DETAILS['exist']}
+						<tr>
+							<td {if $INDEX == 0} class="border-top-0"{/if}>
+								<strong>{$FIELD_DETAILS['name']}</strong> [{$FIELD_NAME}]
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl]: \App\Purifier::encodeHtml

File: layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+							</td>
+							<td {if $INDEX == 0} class="border-top-0"{/if}>
+								<button class="js-download btn btn-sm btn-outline-success" data-prefix="{$FIELD_NAME}" data-js="click | data">
```
Comment (mariuszkrzaczkowski) [inline layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl]: \App\Purifier::encodeHtml

File: modules/Settings/YetiForce/actions/DownloadLanguage.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * YetiForce register action class file.
+ *
+ * @package   Modules
```
Comment (mariuszkrzaczkowski) [inline modules/Settings/YetiForce/actions/DownloadLanguage.php]:
```suggestion
 * @package   Settings
```

File: modules/Settings/YetiForce/actions/DownloadLanguage.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Sławomir Kłos <t.poradzewski@yetiforce.com>
```
Comment (mariuszkrzaczkowski) [inline modules/Settings/YetiForce/actions/DownloadLanguage.php]: ?

File: modules/Settings/YetiForce/views/DownloadLanguageModal.php [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * YetiForce registration modal view class file.
+ *
+ * @package   Modules
```
Comment (mariuszkrzaczkowski) [inline modules/Settings/YetiForce/views/DownloadLanguageModal.php]: ?

File: layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+					{if !$FIELD_DETAILS['exist']}
+						<tr>
+							<td {if $INDEX == 0} class="border-top-0"{/if}>
+								<strong>{\App\Purifier::encodeHtml($FIELD_DETAILS['name'])}</strong>
+								[{\App\Purifier::encodeHtml($FIELD_NAME)}]
+							</td>
+							<td {if $INDEX == 0} class="border-top-0"{/if}>
+								<button class="js-download btn btn-sm btn-outline-success" data-prefix="{\App\Purifier::encodeHtml($FIELD_NAME)}" data-js="click | data">
+									<span class="fas fa-download fa-xs mr-1"></span>
+									{\App\Language::translate('LBL_DOWNLOAD', 'Settings::YetiForce')}
```
Comment (rskrzypczak) [inline layouts/basic/modules/Settings/YetiForce/DownloadLanguageModal.tpl]: check everything from  'Settings::YetiForce'

**Pull request merged**